hardening/oscap/test.py

```diff
@@ -26,6 +26,7 @@
     g.prepare_for_snapshot()
 
 with g.snapshotted():
+    results.report_and_exit()
```

Comment:
Why do you call `results.report_and_exit()` here? Maybe leftover from some testing?

Reply:
Yeah, sorry, I did `git commit -a` :smile: